elements/rh-tabs/rh-tabs.ts

```diff
@@ -0,0 +1,83 @@
+import { html } from 'lit';
+import { customElement, property } from 'lit/decorators.js';
+
+import { classMap } from 'lit/directives/class-map.js';
+import { cascades } from '@patternfly/pfe-core/decorators.js';
+
+import { BaseTabs } from '@patternfly/elements/pf-tabs/BaseTabs.js';
+
+import { RhTab } from './rh-tab.js';
+import { RhTabPanel } from './rh-tab-panel.js';
+
+import { colorContextConsumer, type ColorTheme } from '../../lib/context/color/consumer.js';
+import { colorContextProvider, type ColorPalette } from '../../lib/context/color/provider.js';
+
+import styles from './rh-tabs.css';
+
+export { RhTab };
+
+/**
+ * Tabs
+ */
+@customElement('rh-tabs')
+export class RhTabs extends BaseTabs {
+  static readonly styles = [...BaseTabs.styles, styles];
+
+  static isTab(element: HTMLElement): element is RhTab {
+    return element instanceof RhTab;
+  }
+
+  static isPanel(element: HTMLElement): element is RhTabPanel {
+    return element instanceof RhTabPanel;
+  }
+
+  /**
+   * Sets color theme based on parent context
+   */
+  @colorContextConsumer() private on?: ColorTheme;
+
+  /**
+   * Sets color palette, which affects the element's styles as well as descendants' color theme.
+   * Overrides parent color context.
+   * Your theme will influence these colors so check there first if you are seeing inconsistencies.
+   * See [CSS Custom Properties](#css-custom-properties) for default values
+   */
+  @colorContextProvider()
+  @property({ reflect: true, attribute: 'color-palette' }) colorPalette?: ColorPalette;
+
+  @property({ reflect: true, type: Boolean }) centered? = false;
+
+  // cascade doesn't like undefined values as default
+  @cascades('rh-tab')
+  @property({ reflect: true }) theme?: null | 'base' = null;
+
+  // cascade doesn't like undefined values as default
+  @cascades('rh-tab', 'rh-tab-panel')
+  @property({ reflect: true }) box?: 'box' | 'inset' | null = null;
+
+  @cascades('rh-tab', 'rh-tab-panel')
+  @property({ reflect: true, type: Boolean }) vertical = false;
+
+  @cascades('rh-tab')
+  @property({ attribute: 'border-bottom' }) borderBottom: 'true' | 'false' = 'true';
```

Comment:
does this need to be `'true'|'false'` ? shouldn't it then be a boolean attr?

Reply:
That actually needs to be removed completely.   That is PFE only, and shouldn't have been moved here.  Good catch.